File: lang/php/test/fun/vac/data_structures/LinkedStackTest.php
<?php
    require_once "fun/vac/data_structures/LinkedStack.php";
    require_once "fun/vac/util/TestRunner.php";

    use fun\vac\data_structures\LinkedStack\LinkedStack;
    use fun\vac\util\TestRunner;

    function testLinkedStack(): bool {
        $size = 8192;

        $stack = new LinkedStack();

        for ($i = 1; $i <= $size; $i++) {
            $stack->push($i);
        }

        if ($stack->size() != $size) {
            return false;
        }

        for ($i = $size; $i > 0; $i--) {
            if ($stack->peek() != $i) {
                return false;
            }
            $stack->pop();
        }

        if (!$stack->isEmpty()) {
            return false;
        }

        return true;
    }

    if (count(debug_backtrace()) == 0) {
        TestRunner\parseTest(testLinkedStack());
    }
?>

File: lang/python/test/fun/vac/data_structures/LinkedStackTest.py
from fun.vac.data_structures.LinkedStack import LinkedStack
from fun.vac.util import TestRunner

def testLinkedStack()-> bool:
    size = 8192

    stack = LinkedStack()

    for i in range(1, size + 1):
        stack.push(i)

    if stack.size() != size:
        return False

    for i in range(size, 0, -1):
        if stack.peek() != i:
            return False
        stack.pop()

    return stack.isEmpty()

if __name__ == "__main__":
    TestRunner.parseTest(testLinkedStack())
